aides_logement/tests/tests_calcul_apl_locatif.catala_fr

```diff
@@ -1,10 +1,11 @@
-> Inclusion: ../aides_logement.catala_fr
+> Usage de France
```

Comment:
Why do we need `France` module here?

Reply:
The file uses constructors from the `France.Collectivité` type (`Métropole`, `Mayotte`, etc.)